tests/integration/features/steps/integration_steps.py

```diff
@@ -638,6 +638,36 @@ def _i_cannot_see_the_backup_named_backupname_when_i_list_the_backups(
     assert found is False
 
 
+@then('I cannot see purged backup files for the "{table_name}" table in keyspace "{keyspace}"')
+def _i_cannot_see_purged_backup_files_for_the_tablename_table_in_keyspace_keyspacename(
+    context, table_name, keyspace
+):
+    storage = Storage(config=context.medusa_config.storage)
+    path = os.path.join(
+        storage.prefix_path + context.medusa_config.storage.fqdn, "data", keyspace, table_name
+    )
+    sb_files = len(storage.storage_driver.list_objects(path))
+
+    node_backups = storage.list_node_backups()
+    # Parse its manifest
```

Comment:
It's not what you're doing at this line. This comment belongs to line 656 instead.

Reply:
right! This should be fixed now